tasks/sep/006-2-sc-changes/VALIDATION.md

```diff
@@ -13,6 +13,18 @@ For each contract listed in the state diff, please verify that no contracts or s
 
 The following state overrides should be seen:
 
+### `0xC2Be75506d5724086DEB7245bd260Cc9753911Be` (Superchain Config)
+
+Links:
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/e3004259a4724ad040fe437f219e8e35af391a56/superchain/configs/sepolia/superchain.yaml#L8)
+- [Etherscan](https://sepolia.etherscan.io/address/0xC2Be75506d5724086DEB7245bd260Cc9753911Be)
+
+Enables the simulation by setting the threshold to 1:
```

Comment:
This looks wrong.

Reply:
Oops, deleted in 891122b, since the **Meaning** bullet describes this

Reply:
Tweaked in 702acdd to render better